setup.cfg

```diff
@@ -57,6 +57,7 @@ dev =
 	sphinxcontrib-httpdomain
 	bump2version
 	wheel
+  twine
```

Comment:
I don't think indentation is syntactic in cfg files, but in case it is, the indentation is different here.

Reply:
Meh, tabs vs. spaces 🔥

Reply:
Speaking of bump2version, I think it rewrites this file each time, converting whitespace